src/IoC/Autofac/NuGet/Content/IocBootstrapper.cs.pp

```diff
@@ -1,19 +0,0 @@
-﻿using Kekiri.IoC.Autofac;
-
-namespace $rootnamespace$
```

Comment:
This was laziness on my part. I'm not sure how and if bootstrappers work in a project.json world. If you figure out how to make them work, you can bring this back.